plenum/server/propagator.py

```diff
@@ -81,6 +82,15 @@ def forwarded(self, req: Request) -> bool:
         """
         return self[req.key].forwarded
 
+    def ordered_by_replica(self, request_key):
```

Comment:
Please add/extend tests for this new method

Reply:
Done.